src/ng-dashboard-aggregator/templates/hpa.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
This template and apiVersion is outdated for kubernetes 1.26 onwards:
lets take reference from this hpa.yaml file: https://github.com/harness/helm-platform/blob/main/src/harness-manager/templates/hpa.yaml
with autoscaling values.yaml changes as well

Reply:
Fixed